ticker/src/main/java/com/robinhood/ticker/TickerColumnManager.java

```diff
@@ -74,6 +74,10 @@ void setCharacterLists(String... characterLists) {
         }
     }
 
+    List<char[]> getCharacterLists() {
```

Comment:
Let's add a nullability annotation here? This may also need some javadoc explaining the nullability. What would it mean to get a null list vs an empty list vs a list with an empty array, for instance?

The whole list-of-arrays still seems odd, but might be easier to deal with internally. This change is gonna make it party of the public api, so if we're in doubt at all, now is the time to change it.

Reply:
yeah i dont really like it either, im thinking of adding some kind of interface or wrapper class for it so it'll be something like `List<CharacterList>` or something better named lol.. this should be okay though because this is actually not part of the public API (package private on the column manager).

I also didn't add the nullability annotation here because I didn't want to unnecessarily pull in support annotations in this library just for that. Considering that it's an internal API, i think its ok.

I'll do the list of character list thing before 2.0 release.

Reply:
ah, derp. didn't catch it was package private.